sdk/management/samples/samples.json

```diff
@@ -850,6 +850,12 @@
       "filePath": "azure-samples/src/main/java/com/microsoft/azure/management/appservice/samples/ManageLinuxFunctionAppSourceControl.java",
       "githubPath": "Azure-Samples/app-service-java-configure-deployment-sources-for-functions-on-linux",
       "description": "Getting started on configuring deployment sources for Functions on Linux using Java"
+    },
+    {
+      "owner": "yaohaizh",
```

Comment:
use your alias? or catherine's

Reply:
All owner remaining not change currently, so I think it would be better to change it later in the whole json.